OpenRA.Game/Game.cs

```diff
@@ -481,7 +481,14 @@ static void InnerLogicTick(OrderManager orderManager)
 					Sound.Tick();
 					Sync.CheckSyncUnchanged(world, orderManager.TickImmediate);
 
-					if (world != null)
+					if (world == null)
+						return;
+
+					// Don't tick when the shellmap is disabled
+					// We want the perf graph to continue though
+					if (!world.ShouldTick)
+						PerfHistory.Tick();
```

Comment:
This is ticked inside the `if` and also inside the `else`.  you can move it outside the conditional to avoid the duplication.